Core/include/Acts/Seeding/detail/CylindricalSpacePointGrid.hpp

```diff
@@ -36,22 +37,25 @@ using CylindricalBinnedGroupIterator = Acts::BinnedGroupIterator<
 
 struct CylindricalSpacePointGridConfig {
   // minimum pT to be found by seedFinder
-  float minPt = 0;
+  float minPt = 0 * Acts::UnitConstants::MeV;
   // maximum extension of sensitive detector layer relevant for seeding as
   // distance from x=y=0 (i.e. in r)
-  float rMax = 0;
+  float rMax = 320 * Acts::UnitConstants::mm;
```

Comment:
Default value changes. Is this intentional?

Reply:
It was intentional, meaning the I'm not sure if a default value of `0` makes sense. With a zero you'll also have some FPE when computing the grid binning I'm afraid